src-api/docs/_page.tsx

```diff
@@ -54,7 +71,7 @@ const ApiDocsPage = () => {
 
 	const onClickLogo = (event: ReactMouseEvent) => {
 		event.preventDefault()
-		window.location.assign('/')
+		window.location.assign(ENV__ROOT_URL_PATH + '/') // Forced refresh
```

Comment:
Let's do it the React's way - https://reactrouter.com/en/main/components/link#reloaddocument

Reply:
I was actually wondering if there is already a built-in feature.